CHANGELOG.md

```diff
@@ -19,6 +21,6 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 -   A library to spin up a bitcoind node and do the things we always do: activate segwit, fund addresses, mine blocks.
 
-[Unreleased]: https://github.com/comit-network/comit-rs/compare/0.1.0...HEAD
-
+[Unreleased]: https://github.com/comit-network/comit-rs/compare/0.2.0...HEAD
+[0.2.0]: https://github.com/comit-network/comit-rs/compare/0.1.0...0.2.0
```

Comment:
Haha :see_no_evil:

This still pointing to `comit-rs` :)

Reply:
Well caught :)

Reply:
In case you are using the release action, it relies on you providing a correct URL for the first tag.

Reply:
I'm not :]